SingularityService/src/main/java/com/hubspot/singularity/data/RequestManager.java

```diff
@@ -90,6 +93,7 @@
   );
 
   private final Map<Class<? extends SingularityExpiringRequestActionParent<? extends SingularityExpiringRequestParent>>, Transcoder<? extends SingularityExpiringRequestActionParent<? extends SingularityExpiringRequestParent>>> expiringTranscoderMap;
+  private final ApiCache<String, SingularityRequestWithState> requestsCache;
```

Comment:
shoudl the `getRequests(List<String> requestIds)` and the singular `getRequest(String requestId)` also use the cache? Doesn't look like they are at the moment

Reply:
I didn't update elsewhere yet because I wasn't sure if that's what we wanted to do since we were most concerned by the endpoint to get all requests

Reply:
since it all pulls form the same place, and we are constnatly updating everything, I think it'd be worth it to update. I believe that the individual request endpoint was also pretty high up on the usage. Can always be a follow up PR if we want to check how effective this is first too

Reply:
I added the cache to the getRequests(List<String> requestIds) and the singular getRequest(String requestId), and the singular request has a non-cache flag now for Orion usage.